js/badges.js

```diff
@@ -25,7 +25,16 @@ function badges() {
         var data = "<li id='badgesItem_" + obj.id + "' class='collection-badges'>" +
             "<img src='" + imageDirectory + obj.image + "' alt='" + obj.name + "' title='" + obj.name + "' class='badgesImg'>" +
             "</li>";
-        $("#badgesDisplay").append(data);
+            $("#badgesDisplay").append(data);
+            $("#badge_count").html("(" + count() + "/4)");
```

Comment:
As an improvement idea from my side. Please add a variable with the number of badges to the badges object and use this. https://en.wikipedia.org/wiki/Magic_number_%28programming%29